xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -454,6 +454,7 @@ void CVideoPlayerAudio::Process()
           m_processInfo.SetAudioChannels(audioframe.format.m_channelLayout);
           m_processInfo.SetAudioSampleRate(audioframe.format.m_sampleRate);
           m_processInfo.SetAudioBitsPerSample(audioframe.bits_per_sample);
+          m_processInfo.SetAudioBitRate(m_streaminfo.bitrate);
```

Comment:
this is not the measured bitrate, it is already exposed by GetAudioStreamInfo in SPlayerAudioStreamInfo::bitrate